src/Scene/BoundingBox.js

```diff
@@ -5,32 +5,37 @@
  */
 
 import defaultValue from 'Core/defaultValue';
-import MathExt from 'Core/Math/MathExtented';
+import mE from 'Core/Math/MathExtented';
 import Point2D from 'Core/Math/Point2D';
 import THREE from 'THREE';
 import CoordCarto from 'Core/Geographic/CoordCarto';
 
 /**
  *
- * @param {type} minLongitude : longitude minimum
- * @param {type} maxLongitude : longitude maximum
- * @param {type} minLatitude  : latitude minimum
- * @param {type} maxLatitude  : latitude maximum
+ * @param {type} west : longitude minimum
+ * @param {type} east : longitude maximum
+ * @param {type} south  : latitude minimum
+ * @param {type} north  : latitude maximum
```

Comment:
What about storing 4 floats in a array, and add an additionnal parameter (`unit`?).
This way we could handle *degrees* and *metres* more easily in the codebase.

In the same way, `south` would be a function, returning either `coords[0]` (if `unit` is degrees) or convert `coords[0]` from `unit` to degrees.
Note that for a first step we should __not__ support conversion and simply throw an error.